src/models/form.ts

```diff
@@ -0,0 +1,18 @@
+import { Prop, PropType, Ref } from "@typegoose/typegoose";
+import SignatureChainLink from "./signature-chain-link";
+import FormInstance from "./form-instance";
+
+class Form {
+    //! Still confused on how IDs work here???? Do I need to add one? 😭
+    
+    @Prop({ required: true, type: () => String }) //? Also be unique?
```

Comment:
I have the same comment as department - we can make it unique, but it's probably not a big deal if it isn't